### PR TITLE
fixed the styling for when the screen is 100%

### PR DESCRIPTION
**Pull request by gigicupp**
@masoncpott @danielharmon Hi team! Can you please do a code review on this PR?
I only changed styled components and sizing to make sure things render correctly when screen is 100%. lol

### CODE REVIEW

File: client/src/components/location.jsx [hunk 1]
```diff
@@ -18,19 +18,19 @@ const CardContainer = Styled.div`
   cursor: pointer;
   flex-shrink: 0;
   display: grid;
-  grid-template-columns: 0.2fr 1.5fr 1.5fr 0.7fr;
+  grid-template-columns: 1fr;
```
Comment (danielharmon) [inline client/src/components/location.jsx:21]: What are frs?

File: client/src/components/location.jsx [hunk 2]
```diff
@@ -18,19 +18,19 @@ const CardContainer = Styled.div`
...
+    "img"
+    "company"
+    "country"
+    "footer";
   transition: 0.3s;
-  width: 360px;
-  height: 480px;
-  border-radius: 5%; 
-  margin: 10px;
+  max-width: 260px;
```
Comment (danielharmon) [inline client/src/components/location.jsx:30]: I really like the max-width property

File: client/src/components/material.jsx [hunk 3]
```diff
@@ -100,6 +104,8 @@ const Card = Styled.div`
     margin-right: 10px;
     background: url(${props => props.url});
     z-index: 10;
```
Comment (danielharmon) [inline client/src/components/material.jsx:106]: whats this z-index doing?

**Review by danielharmon — APPROVED** [review]

File: client/src/components/location.jsx (hunk 1 above)
Comment (gigicupp) [reply client/src/components/location.jsx:21]: fractions or free space

File: client/src/components/material.jsx (hunk 3 above)
Comment (gigicupp) [reply client/src/components/material.jsx:106]: I need things to display on top of each other on hover.

**Pull request merged**